site/docs/pages/wallet/types.mdx

```diff
@@ -82,6 +82,21 @@ export type WalletDropdownDisconnectReact = {
 };
 ```
 
+## `WalletDropdownFundLinkReact`
+
+```ts
+export type WalletDropdownFundLinkReact = {
+  className?: string; // Optional className override for the element
+  href: string;
+  icon?: ReactNode;
+  rel?: string;
+  openIn?: 'popup' | 'tab'; // Whether to open the funding flow in a tab or a popup window
+  target?: string; // Where to open the target if `openIn` is set to tab
```

Comment:
I'm a little confused about the `target`. Isn't the only target `https://keys.coinbase.com/funding`?

If they want to have a customizable link then they would use `WalletDropdownLink`

Reply:
Perhaps i'm confusing the target with the src - but still confused about what the target is.

Reply:
https://www.w3schools.com/tags/att_a_target.asp

`target` is a different attribute than `href` - `target` will control where `href` opens (i.e. new tab or replace current tab)

Reply:
`href` is hardcoded here to be keys.coinbase.com/fund